code/controllers/subsystem/alarm.dm

```diff
@@ -0,0 +1,30 @@
+SUBSYSTEM_DEF(alarms)
+   	name = "Alarms"
+   	init_order = INIT_ORDER_ALARMS // 2
+    var/datum/alarm_handler/atmosphere/atmosphere_alarm = new()
+    var/datum/alarm_handler/burglar/burglar_alarm = new()
+    var/datum/alarm_handler/camera/camera_alarm = new()
+    var/datum/alarm_handler/fire/fire_alarm = new()
+    var/datum/alarm_handler/motion/motion_alarm = new()
+    var/datum/alarm_handler/power/power_alarm = new()
+    var/list/datum/alarm/all_handlers
+
+/datum/controller/subsystem/alarms/Initialize(start_timeofday)
+    all_handlers = list(SSalarms.atmosphere_alarm, SSalarms.burglar_alarm, SSalarms.camera_alarm, SSalarms.fire_alarm, SSalarms.motion_alarm, SSalarms.power_alarm)

```

Comment:
THESE ARE SPACES I'M GOING TO SHANK YOU @AffectedArc07

Reply:
I don't get it do you dislike spaces between comas? And the above with equal signs?

Reply:
Nah it's the indenting, it should be done with tabs but it's all fumed up because of the IDE or hitting space at instead